cloudformationTemplates/startChatContactAPI/README.md

```diff
@@ -1,83 +1,137 @@
+    - `contactFlowId` \ `instanceId`: You can find the contact flow id and instance id when viewing a contact flow. For example, if the arn for your flow is 'arn:aws:connect:us-west-2:123456789012:instance/11111111-1111-1111-1111-111111111111/contact-flow/22222222-2222-2222-2222-222222222222', the contact flow id is '22222222-2222-2222-2222-222222222222' and instance id is 11111111-1111-1111-1111-111111111111
+
+        ![Contact Flow Id's](images/updateContactFlowIds.png)
+
+    - Go to your [CloudFormation stack](https://us-west-2.console.aws.amazon.com/cloudformation/home) by clicking on Stacks and then startChatContactAPI on the left hand side. Update the Contact Flow ID and the Instance ID in your cloud formation stack
+    ![Update Cloud Formation Stack](images/updateCloudFormationStack.png)
+    -   These are added as Lambda environment variables.
 
-Note: you will want to show the widget only when there is a conversation in progress. If the widget is visible when there is no conversation, you will just see a loading spinner.
+## Getting started with Pre Built Widget
 
-1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo.
+You can refer to the `widgetIndex.html` file in this repo to see an example of how to use the widget.
 
+1. In your website's html code, import the 'amazon-connect-chat-interface.js' file from this repo. Also import jQuery
     ```html
-    <script src="amazon-connect-chat-interface.js"></script>
+        <script src="amazon-connect-chat-interface.js"></script>
+        <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
     ```
 
-2. Initialize the Chat Interface on page load. Note: you need to update this to include the root id of the div where the customer chat widget will live.
-
-    ```js
-    $(document).ready((a) => {
-      connect.ChatInterface.init({
-        containerId: 'root' // This is the id of the container where you want the widget to reside
-        });
-    });
+2. Create a `div` with id of `root`. 
+    ```html
+    <div id="root">
+    </div>
     ```
+3. Get your API Gateway URL by going to your AWS Console and going to API Gateway product
+
+    ![API Gateway](images/apiGatewayUrl.png) 
 
-3. Start the chat based on a user action. You will want to add fields for the customer name and username because those fields are used in the Lambda function that was created. Note: you need to update this to include the API Gateway endpoint that was created in the CloudFormation stack. To see examples of the success and failure handlers, refer to the [example implementation](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/blob/master/cloudformationTemplates/asyncCustomerChatUX/website/index.html#L283).
+4. Paste the below code as a script section in your html file and ***Update the 4 TODO fields***
 
     ```js
```

Comment:
if you're including a script tag then it should be formatted html so the syntax renders correctly.

You also double indented the script here.

Reply:
Not sure how you find double indentation. I see it as underneath the bullet point
![Screen Shot 2020-04-15 at 7 13 12 PM](https://user-images.githubusercontent.com/9209695/79407054-614f0780-7f4d-11ea-9fc6-c0363c050fdf.png)

Reply:
do you see how where it says
<script>
then there are two lines before $(document)  ? that's double indenting.